build.proj

```diff
@@ -77,6 +79,34 @@
   <UsingTask TaskName="ValidateStrongNameSignatureTask" AssemblyFile="$(LibraryToolsFolder)\Microsoft.WindowsAzure.Build.Tasks.dll"  />
   <UsingTask TaskName="FilterOutAutoRestLibraries" AssemblyFile="$(LibraryToolsFolder)\Microsoft.WindowsAzure.Build.Tasks.dll"  />
   
+  <!-- Building ClientRuntime projects -->
+  <Target Name="BuildClientRuntime">
+      <Exec Command="dotnet restore" WorkingDirectory="%(ClientRuntimeProjects.RootDir)\%(ClientRuntimeProjects.Directory)"/>
+      <Exec Command="dotnet build --configuration $(Configuration)" WorkingDirectory="%(ClientRuntimeProjects.RootDir)\%(ClientRuntimeProjects.Directory)"/>
+  </Target>
+
+  <Target Name="DisableSN" > 
+      <!-- Check for admin privs -->
+      <Exec Command="net session" IgnoreExitCode="true" StandardErrorImportance="Low">
+        <Output PropertyName="isadmin" TaskParameter="ExitCode" />
+      </Exec>
+  
+      <!-- Disable strong name checking -->  
+      <Exec Command='"C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.1 Tools\sn.exe" -Vr *' Condition="$(isadmin) == 0"/>
```

Comment:
>C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.1 Tools [](start = 22, length = 74)

Any reason we want to hardcode this? Consider using $(WindowsSDK_ExecutablePath_x86) and $(WindowsSDK_ExecutablePath_x64) ==> that means we will have to execute %VS140COMNTOOLS%VsDevCmd.bat to get all the environment variable needed.
Alternate way would be to actually check in the tools that our build system needs and depend upon inside the repo (e.g. similar to what we have done for nuget.exe, we can check in sn.exe)